demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -750,16 +750,37 @@ <h5>Simulcast Encoding Parameters</h5>
           title="Toggle video WebRTC stats display" data-bs-toggle="button" aria-pressed="false" autocomplete="off">
           <%= require('../../node_modules/open-iconic/svg/signal.svg') %>
         </button>
-        <button id="button-record-self" type="button" class="btn btn-success mx-1 mx-xl-2 my-2"
-          title="Record my audio and video" data-bs-toggle="button" aria-pressed="false" autocomplete="off">
-          <svg class="svg">
-            <circle id="rec-svg" cx="10" cy="10" r="8" fill="red"></circle>
-          </svg>
-        </button>
-        <button id="button-record-cloud" type="button" class="btn btn-success mx-1 mx-xl-2 my-2"
-          title="Capture audio and video in the cloud" data-bs-toggle="button" aria-pressed="false" autocomplete="off">
-          <%= require('../../node_modules/open-iconic/svg/cloud-upload.svg') %>
-        </button>
+        <div class="btn-group mx-1 mx-xl-2 my-2" role="group" aria-label="Select destination">
```

Comment:
okay I will add the screenshot

Reply:
Nits on the UX:
* Lets use the record icon rather then cloud icon, I think that is more indicative of the end goal.
* Lets not bother with the icons in the drop down since that messes up the padding and we don't do that in the rest of the dropdowns.
* Just dropping the URL in a random `<p>` is not ideal. Maybe try adding it to the area with the attendee and call ID.